cypress/features/regression/stepSequenceItem.feature

```diff
@@ -15,8 +15,8 @@ Feature: Step Sequence Item component
 
   @positive
   Scenario Outline: I set status to <status>
-    When I open default "Step Sequence Item" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
-    Then text "Step Label" color is set to "<color>"
+    When I open step-sequence-item "Step Sequence Test" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
```

Comment:
this TC needs to be removed, as we are checking colours in `chromatic` also delete please the corresponding json objects from `cypress/fixtures/commonComponents/stepSequence.json`